<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgVideoEdit</class>
 <widget class="QDialog" name="DlgVideoEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cut video</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Properties for all shots</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="wgt_QVideoPlayer" name="VideoPlayer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Start :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="SeekLeftBt">
           <property name="toolTip">
            <string extracomment="Tooltip in dialog box">Move to start position</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/player_rew.png</normaloff>img/player_rew.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="DefStartPosBT">
           <property name="toolTip">
            <string extracomment="Tooltip in dialog box">Use current position as start position</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/PosStart.png</normaloff>img/PosStart.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="StartPosEd">
           <property name="minimumSize">
            <size>
             <width>105</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>105</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string extracomment="Tooltip in dialog box">Define the start position to use for</string>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="specialValueText">
            <string notr="true"/>
           </property>
           <property name="displayFormat">
            <string notr="true">HH:mm:ss.zzz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>End :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="EndPosEd">
           <property name="minimumSize">
            <size>
             <width>105</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>105</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string extracomment="Tooltip in dialog box">Define the end position to use for</string>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="specialValueText">
            <string notr="true"/>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::HourSection</enum>
           </property>
           <property name="displayFormat">
            <string notr="true">HH:mm:ss.zzz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="DefEndPosBT">
           <property name="toolTip">
            <string extracomment="Tooltip in dialog box">Use current position as end position</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/PosEnd.png</normaloff>img/PosEnd.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="SeekRightBt">
           <property name="toolTip">
            <string extracomment="Tooltip in dialog box">Move to end position</string>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/player_fwd.png</normaloff>img/player_fwd.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Duration :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ActualDuration">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>21</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string notr="true">00:00:00.000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Volume :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="VolumeReductionFactorCB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Define the &quot; Master Volume &quot; for the movie</string>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKBT">
       <property name="toolTip">
        <string>Accept modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/action_ok.png</normaloff>img/action_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseBT">
       <property name="toolTip">
        <string>Cancel modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/action_cancel.png</normaloff>img/action_cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HelpBT">
       <property name="toolTip">
        <string>Open help page describing this dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/help.png</normaloff>img/help.png</iconset>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wgt_QVideoPlayer</class>
   <extends>QWidget</extends>
   <header>wgt_QVideoPlayer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
